lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -24,51 +33,59 @@ def self.replace_version_number(previous_version_number, new_version_number, fil
         end
       end
 
+      def self.determine_next_version_using_labels(repo_name, github_token, rate_limit_sleep)
+        old_version = Actions.sh("git describe --tags --abbrev=0").strip
+        UI.important("Determining next version after #{old_version}")
+        org = "RevenueCat"
+
+        commits = get_commits_since_old_version(org, github_token, old_version, repo_name)
```

Comment:
Hmm I'm wondering if we should move all of this code to a different helper, since `RevenueCatInternalHelper` is growing pretty big... Maybe a `ChangelogHelper`, or `GithubHelper`?  I created a second helper for the `update_hybrids_versions_file` action, in case that helps.